src/components/PopupDeleteCard.js

```diff
@@ -0,0 +1,21 @@
+import Popup from "./Popup.js";
+
+class PopupDeleteCard extends Popup {
```

Comment:
This class should be named `PopupWithConfirmation`, since it's a general class, which can be used for any confirmation, not only card delete confirmation